### ISSUE TITLE
Suggestion: Move Examples directory, and related code, to a separate repository

### ISSUE THREAD

**Issue by cmathew** [pusher/pusher-angular#21]
Not optimal that, for a library which is only ~400 lines, we must download somewhere around ~90K in the Example and Public directories.

Once again, thank you for this wonderful project!

**Comment by jackfranklin** [comment]
@cmathew good point - rather than move the examples out could we also configure Bower to not download them? That would work too I think.

**Comment by cmathew** [comment]
@jackfranklin yes, some additions to the "ignore" section of `bower.json` would work as well!
